modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountUserDisplay.java

```diff
@@ -99,6 +102,27 @@ public String getAccountRoleNames(long accountEntryId, Locale locale)
 			ListUtil.sort(accountRoleNames), StringPool.COMMA_AND_SPACE);
 	}
 
+	public String getAccountValidDomains() {
+		List<AccountEntryUserRel> accountEntryUserRels =
+			_getAccountEntryUserRels(getUserId());
+
+		return Stream.of(
+			accountEntryUserRels
+		).flatMap(
+			List::stream
+		).map(
+			AccountEntryUserRelModel::getAccountEntryId
+		).map(
+			AccountEntryLocalServiceUtil::fetchAccountEntry
+		).filter(
+			Objects::nonNull
+		).map(
+			AccountEntry::getDomains
+		).collect(
+			Collectors.joining(StringPool.COMMA)
+		);
+	}
```

Comment:
We should make a call to [`.distinct()`](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#distinct--) as well, otherwise we end up with repeated domains if more than one account has the same domain.

<img width="696" alt="Screen Shot 2020-06-12 at 5 38 07 PM" src="https://user-images.githubusercontent.com/6403097/84551516-85bc2b80-acd3-11ea-8125-fa486da680bb.png">